### ISSUE TITLE
support for mobile touch?

### ISSUE THREAD

**Issue by ldenoue** [kiwphi/pop-shoot#15]
Would be great to handle touchEvents in addition to mousedown and mouseup...
Laurent

**Comment by kiwphi** [comment]
I've thought of it before, but the game being very fast-paced and requiring a lot of precision makes it not suited for mobile (touch controls are not accurate enough, and small screens don't help either when there's too many bullets on-screen). The only way to make it playable on mobile is to dumb it down which I'm not willing to do. Maybe someone would like to fork-it and make it mobile friendly...
